### PR TITLE
feat(typescript): move typescript variable instances from protractor/…

### LINKED ISSUE #3564: TypeScript changes - change the `protractor/globals` import to just `protractor`
As discussed offline:

To simplify the story for TypeScript users and eliminate confusion about globals, export global singletons from `protractor` instead of `protractor/globals`.

Before:

```ts
import {browser, element} from 'protractor/globals';
import {ElementFinder} from 'protractor';

describe('my app', () => {
  myElement: ElementFinder;

  beforeEach(() => {
    browser.get('example.com');
    myElement = element(by.css('foo'));
  });
});
``` 

After

```ts
import {browser, element, ElementFinder} from 'protractor';

describe('my app', () => {
  myElement: ElementFinder;

  beforeEach(() => {
    browser.get('example.com');
    myElement = element(by.css('foo'));
  });
});
``` 

The configuration option `noGlobals` will continue to do the same thing - if `true`, Protractor will only define `global.protractor`, otherwise, it will define all of `global.protractor, global.browser, global.element, global.by, global.$, global.$$`. Whether it is enabled or not should have no effect on TS users.

### PR DESCRIPTION
**Pull request by juliemr**
…globals to protractor

Breaking change for TypeScript:

Instead of importing globals like `browser` from `protractor/globals`, import from `protractor`.

Before:

```ts
import {browser, element} from 'protractor/globals';
import {ElementFinder} from 'protractor';

describe('my app', () => {
  myElement: ElementFinder;

  beforeEach(() => {
    browser.get('example.com');
    myElement = element(by.css('foo'));
  });
});
```

After

```ts
import {browser, element, ElementFinder} from 'protractor';

describe('my app', () => {
  myElement: ElementFinder;

  beforeEach(() => {
    browser.get('example.com');
    myElement = element(by.css('foo'));
  });
});
```

Closes #3564

### CODE REVIEW

File: spec/install/typescript_spec.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import {browser, by, By, element, $, $$, ExpectedConditions, protractor} from 'protractor/globals';
-import {ProtractorBrowser} from 'protractor';
+import {browser, by, By, element, $, $$, ExpectedConditions, protractor, ProtractorBrowser} from 'protractor';
```
Comment (cnishina) [inline spec/install/typescript_spec.ts]: Reviewing this spec, it probably does not need `ProtractorBrowser`... lets remove it.

**Comment by juliemr** [comment]
Done! New github review, exciting.

**Comment by ccrowhurstram** [comment]
this was a breaking change - why wasn't the major version bumped?

or maybe protractor is not following semver? That's fine of course, but it would be good to know so that I can specify exact version in my `devDependencies`

**Comment by cnishina** [comment]
Hi @ccrowhurstram. Protractor TypeScript support is still experimental. The breaking change was listed in the changelog.

**Comment by ccrowhurstram** [comment]
Thanks for getting back to me.

So @cnishina, aside from the typescript declarations, Protractor is following semver?

**Comment by cnishina** [comment]
Yes. We do follow semver.

**Comment by ccrowhurstram** [comment]
OK, thanks

**Pull request merged**